### ISSUE TITLE
div as parents of dt/dd

### ISSUE THREAD

**Issue by honoka-h** [dequelabs/axe-core#1289]
Expectation: 
dt/dd elements below div element passes the axe-core checks.
```html
<dl>
<div>
<dt>dt element inside div</dt>
<dd>dd element inside div</dd>
</div>
</dl>
```

Actual:
The above is reported as an issue:
>Ensures \<dt\> and \<dd\> elements are contained by a \<dl\>

Motivation:
I have seen that this issue has been taken care of in #262, but the issue is still reported.
Could you please look into it?

axe-core version: 3.1.1

Browser and Assistive Technology versions
Chrome 71.0.3578.98

**Comment by kensgists** [comment]
This issue exists in axe-core 3.1.2, as well.

**Comment by marcysutton** [comment]
This is already in the works in a PR: https://github.com/dequelabs/axe-core/pull/1284

Closing as a duplicate of https://github.com/dequelabs/axe-core/issues/1265
